DapperDoodle/Case.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Globalization;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace DapperDoodle
+{
+    public enum Case
+    {
+        IgnoreCase,
+        Uppercase,
+        PascalCase,
+        Lowercase,
+        CamelCase,
+        KebabCase,
+        SnakeCase
+    }
+    
+    public static class CaseHelpers
+    {
+        /// <summary>
+        /// The ConvertCase method currently only supports conversions from Pascal Case to everything else. Other support can later be added.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <param name="casing"></param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentNullException"></exception>
+        public static string ConvertCase(this string value, Case casing)
+        {
+            switch (casing)
+            {
+                case Case.Lowercase:
+                    return value.ToLowerInvariant();
+                case Case.Uppercase:
+                    return value.ToUpperInvariant();
+                case Case.CamelCase:
+                    return char.ToLowerInvariant(value[0]) + value.Substring(1);
+                case Case.PascalCase:
+                {
+                    var textInfo = new CultureInfo("en-Us", false).TextInfo;
+                    return textInfo.ToTitleCase(value);
+                }
+                case Case.IgnoreCase:
+                    return value;
+                case Case.KebabCase:
+                {
+                    if (string.IsNullOrEmpty(value))
+                    {
+                        return value;
+                    }
+                    return Regex.Replace(value,
+                            "(?<!^)([A-Z][a-z]|(?<=[a-z])[A-Z])",
+                            "-$1")
+                        .Trim()
+                        .ToLower();
+                }
+                case Case.SnakeCase:
+                {
+                    if(value is null)
+                        throw new ArgumentNullException(nameof(value));
+                    if (value.Length < 2)
+                        return value;
+                    var sb = new StringBuilder();
```

Comment:
minor: suggest using `string.Join` and a list over `StringBuilder` as the former is more performant. `StringBuilder` should be forgotten (:

Reply:
Thanks. Will do that. 👍🏼